packages/fx-core/resource/yaml-schema/v1.8/yaml.schema.json

```diff
@@ -1972,6 +1974,80 @@
           }
         }
       }
+    },
+    "devChannelCreate": {
```

Comment:
Will you update wiki page to add these actions?

Reply:
Yes, tracked in this task: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/32019035

After the first demo, the action name may get changed. So I will update the wiki when it's finalized.